elasticreindexer/README.md

```diff
@@ -1,15 +1,88 @@
 # Elastic reindexer tool
 
-This tool is able to reindex all mappings and data from an instance of Elasticsearch to another.
+- This tool can be used to copy an Elasticsearch  index/indices from a cluster to another. It can also copy the indices 
+mappings ( we recommend to use it without copying the mappings ). The indices mappings part can be done with the `indices-creator` tool.
+Also, with the `indices-creator` tool some properties of the indices can be changed ( depending on the needs ).
+
+- The main scope of these tools is to copy all the information from an Elasticsearch cluster to another ( all the indices that were
+populated by the Elrond nodes from the genesis until the current time ).
 
 ## How to use
+### STEP 1
+- In order to reindex all the information from an Elasticsearch cluster to another, one has to create all the indices mappings. 
+To do that one has to use the `indices-creator` tool:
+
+    ```
+    cd elasticreindexer
+    cd cmd/indices-creator/
+    go build 
+    ```
+  
+- After the build is done one has to update the `config/cluster.toml` file with the information about the Elasticsearch cluster. In the `cluster.toml` one has to set the URL 
+of the Elasticsearch cluster, and what indices to be used to create the mappings.
+
+- Optionally, the mappings can be also customized ( open file with mappings for every index and set different settings, based on the needs)
+
+- Run `./indices-creator` in order to create all the indices and mappings
+
+_**note:** STEP 1 can be skipped for the clusters that already have the information indexed_ 
+
+***
+
+### STEP 2
+- After the mappings and indices were created we can start to copy all the information. In order to do this one haS to use the `elasticreindexer` tool that 
+will copy every index one by one.
+
+    ```
+    cd elasticreindexer
+    cd cmd/elasticreindexer/
+    go build 
+    ```
+- Update the Elasticsearch instance configuration for both source and destination inside `config.toml`. In the `config.toml` file one has to set the 
+URL of the Elasticsearch `input` instance ( the one from where we have to copy all the information ) and the `output` instance ( the one where all the information 
+from the `input` instance will be copied )
+
+- The `config.toml` file contains by default all the Elasticsearch indices that are populated by an Elrond observing-squad
+
+- Also, if you want to copy indices with timestamp you have to set the `blockchain-start-time` in the `config.toml` file ( by default is the one from the mainnet)
```

Comment:
add more data:
Instances that include a timestamp are already defined in the configuration file. Their cloning will be much faster due to the parallel execution on batches split depending on timestamp

Reply:
added